### ISSUE TITLE
Add option to save screens

### ISSUE THREAD

**Issue by stefandrissen** [simonowen/simcoupe#21]
[spectrumcomputing.co.uk](https://spectrumcomputing.co.uk/forums/viewtopic.php?p=36962#p36962) is now accepting SAM Coupé screen shots. Since they are for preservation an 'original data' format is requested, this seems to be a flat 24k for screen + 16 bytes for palette. No line interrupts or mode switching.

[ASCD](http://www.keprt.cz/progs/ascd/) can export these files, but I am too used to SimCoupe to be able to use ASCD comfortably. Could this be added to SimCoupe?

The library used for processing images is https://github.com/moroz1999/zx-image

**Comment by simonowen** [comment]
It sounds pretty simple so I'll look at adding that in this week.

There were questions about the format for other screen modes, but using contiguous blocks of display data + palette should be good for all I think.

**Comment by simonowen** [comment]
Added in 270dd77915c501cff2c6743ec941d07e4c6ec7bd. See [ssxview.py](https://github.com/simonowen/ssxview) for format details and a Python script to view the output files.

The MODE 4 output should match the existing SS4 files. I'm not sure if SS3 used a 4-entry or 16-entry CLUT, or whether SS1 and SS2 files existed at all.

The new files have unique sizes so I've used a new .ssx file extension to cover them all, plus any format differences. There's also a raw version of the file that supports raster-level effects, which is used if required. This is likely to require some changes in @moroz1999's library.
